database/migration/V20170204_1037__createEventsTable.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE `events` (
```

Comment:
Probably, but it seems like the other database entries have done what I've done (splitting each into a different file).